src/select-project.html

```diff
@@ -0,0 +1,72 @@
+
+<link rel="import" href="shared-styles.html">
+<link rel="import" href="../bower_components/vaadin-combo-box/vaadin-combo-box.html">
+<link rel="import" href="../bower_components/iron-ajax/iron-ajax.html">
+
+
+<dom-module id="select-project">
+  <template>
+    <style>
+    </style>
+
+    <!-- Fetch an array of users to be shown in the grid -->
+    <iron-ajax id="projects" url="/dummy/projects.json" handle-as="json" last-response="{{projects}}"></iron-ajax>
+
+    <vaadin-combo-box filtered-items="[[items]]" label="Project" item-label-path="title" on-filter-changed="_projectsFilter" selected-item="{{selectedProject}}">
+      <template>
+        [[item.title]]
+      </template>
+    </vaadin-combo-box>
+
+  </template>
+
+  <script>
+    class SelectProject extends Polymer.Element {
+      static get is() { return 'select-project'; }
+
+      static get properties () {
+        return {
+          selectedProject: {
+            type: Object,
+            notify: true
+          },
+          items: {
+            type: Array,
+            value: []
+          },
+          projects: {
+            type: Array,
+            value: []
+          }
+        }
+      }
+
+      constructor () {
+        super();
+      }
+
+      ready () {
+        super.ready();
+        this._fetchProjects();
+      }
+
+      _fetchProjects () {
```

Comment:
Always try to favor Polymer observers instead of doing stuff inside the lifecycle functions – it's more reliable in general. In this case, I would add `auto` to the `<iron-ajax>` which will make it automatically fire the request when `url` or other properties are changed and add a observer for the `last-response` property for setting the default value for `selectedProject`.

So instead of
```js
   ready () {
      super.ready();
      this._fetchProjects();
    }

    _fetchProjects () {
      var projectsRequest = this.$.projects.generateRequest();
        
      projectsRequest.completes.then((function () {
        this.selectedProject = this.projects[0];
      }).bind(this));
   }
```
I would have:
```js
      static get observers() {
        return ['_projectsChanged(projects)'];
      }

      _projectsChanged(projects) {
        this.selectedProject = projects[0];
      }
```
together with:
```html
    <iron-ajax id="projects" url="/dummy/projects.json" handle-as="json" last-response="{{projects}}" auto></iron-ajax>
```